### DISCUSSION TITLE
Any way to access location from widget?

### DISCUSSION THREAD

**Discussion by aekanshpartani** [chatwoot/chatwoot#1930] [Q&A]
Is there any way either IP Address or Browser location Enable through which I can access user's location? Need this to show something in nearby area to user.
I tried looking for IP address but not able to find it. Can someone guide me here?

**Comment by sojan-official** [comment]
@aekanshpartani you can enable IP tracking in chatwoot:  https://www.chatwoot.com/docs/self-hosted/enable-ip-logging 
let me know if this helps ?

**Reply by aekanshpartani** [reply]
Thank You @sojan-official . Will look into it and let you know.

**Reply by aekanshpartani** [reply]
Thanks @sojan-official It worked.  Able to get IP Address.
